tests_new/integration_tests/modules/notifications/test_notifications.py

```diff
@@ -11,32 +12,28 @@
 )
 
 
-def test_list_notification(client1, session_share_1):
-    response = list_notifications(client1)
-    assert_that(response).is_not_none()
-    assert_that(response.nodes).is_not_empty()
-    assert_that(response.nodes[0].notificationUri).is_not_none()
-
+def test_list_notification(client1):
+    assert_that(list_notifications(client1)).is_not_none()
     assert_that(list_notifications(client1, filter={'read': True})).is_not_none()
     assert_that(list_notifications(client1, filter={'unread': True})).is_not_none()
     assert_that(list_notifications(client1, filter={'archived': True})).is_not_none()
 
 
-def test_count_unread_notification(client1, session_share_1):
+def test_count_unread_notification(client1):
     assert_that(count_unread_notificiations(client1)).is_greater_than_or_equal_to(0)
 
 
```

Comment:
countReadNotifications and countDeletedNotifications are not used in the frontend. Maybe we should delete them from the backend and remove the tests for countReadNotifications. wdyt?

Reply:
deleteNotification is also not used in the frontend

Reply:
removed all 3 ops